### DISCUSSION TITLE
Isn't showing data type in Rust like before.

### DISCUSSION THREAD

**Discussion by ShouvikXIV** [VonHeikemen/lsp-zero.nvim#295] [Q&A]
So, in Rust whenever I declare a variable, it used to show the data type beside the variable like this =>i32 but for some reason it is not showing it any more. I am a beginner to rust and nvim as well.. Someone please help?

**Comment by VonHeikemen** [comment]
Can you give more details about your setup? It sounds like you are talking about inlay hints, which is a feature that is not available in the latest stable version of Neovim.

**Reply by ShouvikXIV** [reply]
I'm using nvim 0.9.1. Idk it was working last week as well..
